src/actions/requests.rs

```diff
@@ -562,6 +576,10 @@ impl<'a> RequestAction<'a> for Deglob {
     }
 }
 
+/// Execute a command within the workspace.
+///
+/// These are *not* shell commands, but RLS commands. Currently, only the
+/// "rls.applySuggestion" command is supported.
```

Comment:
These are Client commands rather than server commands, but they are handled by the server.